values-4.10-datacenter.yaml

```diff
@@ -6,8 +6,8 @@ clusterGroup:
     acm:
       name: advanced-cluster-management
       namespace: open-cluster-management
-      channel: release-2.5
-      csv: advanced-cluster-management.v2.5.2
+      channel: release-2.6
+      #csv: advanced-cluster-management.v2.5.2
```

Comment:
Let's put 2.6.1 in the comment so there is less chance of errors when uncommenting?

Reply:
Shouldn't this change just go in values-datacenter.yaml . I would retroactively reduce the values-4.10-datacenter.yaml and values-4.11-datacenter.yaml to the minimum divergence needed from values-datacenter.yaml. See how Johnny did it in https://github.com/hybrid-cloud-patterns/medical-diagnosis/ (those files only contain the strict minimum needed)

Reply:
@mbaldessari I went through all the values files and moved the subscriptions around.  The datacenter one is probably the one that has the most subscriptions.  The development subscriptions were moved to the values-4.{10,11}-datacenter.yaml. The "prod" subscriptions were left in the values-datacenter.yaml.  On the factory side it appears that all the subscriptions go into the values-factory.yaml and the values-4.{10,11}-factory.yaml files are empty.